mucommander-os-api/src/main/java/com/mucommander/desktop/ActionShortcuts.java

```diff
@@ -138,11 +138,11 @@ public KeyStroke getDefaultKeystroke(ActionType actionId) {
         case OpenInNewTab:
             return KeyStroke.getKeyStroke(KeyEvent.VK_ENTER, KeyEvent.CTRL_DOWN_MASK);
         case OpenInOtherPanel:
-            return KeyStroke.getKeyStroke(KeyEvent.VK_O, KeyEvent.CTRL_DOWN_MASK);
+            return KeyStroke.getKeyStroke(KeyEvent.VK_O, KeyEvent.SHIFT_DOWN_MASK | KeyEvent.CTRL_DOWN_MASK | KeyEvent.ALT_DOWN_MASK);
```

Comment:
I prefer not to change existing shortcuts as much as possible since users may have got used to the current configuration. in the long term, I think we would drop the terminal action in favor of a terminal widget below the file table and then we could use CMD+R / CTRL+R for switching the focus to that component (and getting rid of the run-command dialog)
we can use this shortcut (SHIFT+CTRL+ALT+O) to the terminal action in the meantime

Reply:
Since all the commanders I know (that have terminal command) use Ctrl/CMD+O, I would suggest to follow this pattern.

Reply:
@pskowronek I see your point but it will place us in a not-that-great place if we get complains about "suddenly" changing the shortcut of 'open in other panel'. I also don't think it matters that much - as you've noticed, @Andreas0602 that I believe is the first user that uses this feature has already configured a different shortcut (F10) for it, and in the long term I'd like to drop that action when we'll have an integrated widget..